modules/tinymce/src/core/demo/html/full_demo.html

```diff
@@ -12,7 +12,7 @@ <h1>Tinymce full featured Page</h1>
     <div id="ephox-ui">
       <textarea></textarea>
     </div>
-    <div class="tinymce">Inline demo</div>
+    <div class="tinymce"><table><tbody><tr><td>Inline demo</td><td>Cell 2</td></tr></tbody></table></div>
```

Comment:
Why was this changed?

Reply:
This was not meant to be committed... thanks for the catch!

- [ ] Revert this

Reply:
Reverted